### ISSUE TITLE
SMBIOS = MacMini8,1?

### ISSUE THREAD

**Issue by gatorback** [hackintosh-efi/DELL-3090MFF-OpenCore#1]
Greetings from Florida!   Did you use SMBIOS=MacMini8,1?  I have been trying for many months to configure 4K for the 3090MFF i3-10105T.  https://www.tonymacx86.com/threads/is-there-any-reason-monterey-will-not-boot-on-a-optiplex-mff-3090-i3-10105.321903/

**Comment by WuLongMiTaoLaiYiDa** [comment]
u can use macmini 8.1 also imac 20.1

**Comment by WuLongMiTaoLaiYiDa** [comment]
don't forget use RU.efi replace sasetup

**Comment by Crazypete76** [comment]
Good day. 

I am unable to even start the installation on my 3090. I am battling with the RU.efi where I can access the suggested settings but cannot edit them. Please assist

**Comment by gatorback** [comment]
@Crazypete76   Did you "remote update" the BIOS?" and change the BIOS settings to turn off the unnecessary items as prescribed in GitHub?  

**CRITICAL:** Did you set DVMT=0x02?   To READ DVMT I used modgrubshell.efi command line:

`setup_var_cv SaSetup 0xF5 0x01`

To write DVMT from the modgrubshell.efi command line:

`setup_var_cv SaSetup 0xF5 0x01 0x02
`

modgrubshell: https://github.com/datasone/grub-mod-setup_var/releases

This is not the forum for assistance: instead go to discord for real-time help assistance: https://dortania.github.io/forums/. There are plenty of users that know much more than me and provide better assistance

I am sure the RU.efi works, however, discord users directed me to use modgrubshell.efi command line and it solved the 4K video problem.   I thought I had changed DVMT, but did not change it to 0x02.  If DVMT = 0x01 and you do not patch the framebuffer , then I think the video will not work.   The robust solution is to set DVMT=0x02
